Client/Frontend/Browser/Playlist/PlaylistPopoverView.swift

```diff
@@ -36,19 +36,18 @@ struct PlaylistPopoverView: View {
                 Text(Strings.PlayList.playlistPopoverAddTitle)
                     .foregroundColor(Color(UIColor.primaryButtonTint))
                     .font(.title2.weight(.medium))
+                    .padding(.bottom, 20.0)
             } else {
                 Text(Strings.PlayList.playlistPopoverAddTitle)
                     .foregroundColor(Color(UIColor.primaryButtonTint))
                     .font(.headline.weight(.medium))
+                    .padding(.bottom, 20.0)
             }
             
-            Spacer(minLength: 20.0)
-            
             Text(Strings.PlayList.playlistPopoverAddBody)
                 .foregroundColor(Color(UIColor.braveLabel))
                 .font(.body)
-            
-            Spacer(minLength: 20.0)
+                .padding(.bottom, 20.0)
```

Comment:
Use container spacing instead of adding padding to all views

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4508/commits/9f72c586778075cc4dc78b26127c110ec6ce4bb8